files/gridlex/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Laurent G."
+github = "https://github.com/devlint/gridlex"
+homepage = "http://gridlex.devlint.fr"
+description = "Gridlex is just a CSS Flexbox Grid System "
+mainfile = "dist/gridlex.min.js"
```

Comment:
Again you're right, there is no JS files, I've made it too quickly.
For the dist/ it's because I've a `src` and a `dist` folders where the min.css file is. So for you I only need to target directly `gridlex.min.css` and not `dist/gridlex.min.css`?
Thank you!